pwiz_tools/Skyline/FileUI/ImportTransitionListColumnSelectDlg.cs

```diff
@@ -386,8 +611,57 @@ private void ComboChanged(object sender, EventArgs e)  // CONSIDER(bspratt) no c
                 if (columns.ProductColumn == comboBoxIndex) columns.ProductColumn = -1;
                 if (columns.ProteinColumn == comboBoxIndex) columns.ProteinColumn = -1;
                 if (columns.FragmentNameColumn == comboBoxIndex) columns.FragmentNameColumn = -1;
-                // Commented out for consistency because there is no product charge column
-                // if (columns.PrecursorChargeColumn == comboBoxIndex) columns.PrecursorChargeColumn = -1;
+                if (columns.ExplicitRetentionTimeWindowColumn == comboBoxIndex)
```

Comment:
This is getting pretty lengthy, I wonder if there's any way use reflection to handle this automatically for each columns.*Column.